address;source_location;insn;indent
0x501ae1;/usr/local/src/conda/python-3.9.19/Python/errors.c:324;MOV	(%RBX),%RBP;
0x501ae4;/usr/local/src/conda/python-3.9.19/Python/errors.c:325;TEST	%RBP,%RBP;
0x501ae7;:0;JE	432076 <_PyErr_NormalizeException.cold+0xa4>;
0x501aed;/usr/local/src/conda/python-3.9.19/Python/errors.c:331;MOV	(%R12),%R13;
0x501af1;/usr/local/src/conda/python-3.9.19/Python/errors.c:335;TEST	%R13,%R13;
0x501af4;:0;JNE	501b03 <_PyErr_NormalizeException+0x63>;
0x501af6;/usr/local/src/conda/python-3.9.19/Include/object.h:408;INCQ	0x23a163(%RIP);
0x501afd;/usr/local/src/conda/python-3.9.19/Python/errors.c:336;MOV	$0x73bc60,%R13D;
0x501b03;/usr/local/src/conda/python-3.9.19/Include/object.h:628;MOV	0x8(%RBP),%RAX;
0x501b07;/usr/local/src/conda/python-3.9.19/Python/errors.c:343;TESTB	$-0x80,0xab(%RAX);
0x501b0e;:0;JE	501c66 <_PyErr_NormalizeException+0x1c6>;
0x501b14;/usr/local/src/conda/python-3.9.19/Python/errors.c:343;TESTB	$0x40,0xab(%RBP);
0x501b1b;:0;JE	501c66 <_PyErr_NormalizeException+0x1c6>;
0x501b21;/usr/local/src/conda/python-3.9.19/Python/errors.c:347;MOV	0x8(%R13),%RDX;
0x501b25;/usr/local/src/conda/python-3.9.19/Python/errors.c:347;TESTB	$0x40,0xab(%RDX);
0x501b2c;:0;JNE	501c3a <_PyErr_NormalizeException+0x19a>;
0x501b32;/usr/local/src/conda/python-3.9.19/Python/errors.c:93;CMP	$0x73bc60,%R13;
0x501b39;:0;JE	501bf8 <_PyErr_NormalizeException+0x158>;
0x501b3f;/usr/local/src/conda/python-3.9.19/Include/object.h:628;MOV	0x8(%R13),%RSI;
0x501b43;/usr/local/src/conda/python-3.9.19/Python/errors.c:96;TESTB	$0x4,0xab(%RSI);
0x501b4a;:0;JNE	501c79 <_PyErr_NormalizeException+0x1d9>;
0x501b50;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:185;MOV	%R13,0x28(%RSP);
0x501b55;/usr/local/src/conda/python-3.9.19/Include/internal/pycore_pystate.h:55;MOV	0x2463fc(%RIP),%R11;
0x501b5c;/usr/local/src/conda/python-3.9.19/Include/internal/pycore_pystate.h:82;TEST	%R11,%R11;
0x501b5f;:0;JE	431fd2 <_PyErr_NormalizeException.cold>;
0x501b65;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:73;MOV	0x8(%RBP),%RDI;
0x501b69;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:74;TESTB	$0x8,0xa9(%RDI);
0x501b70;:0;JE	501b84 <_PyErr_NormalizeException+0xe4>;
0x501b72;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:81;MOV	0x38(%RDI),%R8;
0x501b76;:0;MOV	(%RBP,%R8,1),%RAX;
0x501b7b;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:114;TEST	%RAX,%RAX;
0x501b7e;:0;JNE	501c8e <_PyErr_NormalizeException+0x1ee>;
0x501b84;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:116;MOV	$0x1,%ECX;
0x501b89;:0;XOR	%R8D,%R8D;
0x501b8c;:0;LEA	0x28(%RSP),%RDX;
0x501b91;:0;MOV	%RBP,%RSI;
0x501b94;:0;MOV	%R11,%RDI;
0x501b97;:0;CALL	4f0370 <_PyObject_MakeTpCall>;
0x501b9c;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:116;MOV	%RAX,%RCX;
0x501b9f;/usr/local/src/conda/python-3.9.19/Python/errors.c:103;TEST	%RCX,%RCX;
0x501ba2;:0;JE	501cfc <_PyErr_NormalizeException+0x25c>;
0x501ba8;/usr/local/src/conda/python-3.9.19/Python/errors.c:103;MOV	0x8(%RCX),%RSI;
0x501bac;/usr/local/src/conda/python-3.9.19/Python/errors.c:103;TESTB	$0x40,0xab(%RSI);
0x501bb3;:0;JE	501ccf <_PyErr_NormalizeException+0x22f>;
0x501bb9;/usr/local/src/conda/python-3.9.19/Include/object.h:422;DECQ	(%R13);
0x501bbd;:0;JE	432082 <_PyErr_NormalizeException.cold+0xb0>;
0x501bc3;/usr/local/src/conda/python-3.9.19/Python/errors.c:377;MOV	%RBP,(%RBX);
0x501bc6;/usr/local/src/conda/python-3.9.19/Python/errors.c:378;MOV	%RCX,(%R12);
0x501bca;/usr/local/src/conda/python-3.9.19/Python/errors.c:379;MOVZX	0x24(%R15),%EBP;
0x501bcf;/usr/local/src/conda/python-3.9.19/Python/errors.c:379;DEC	%EBP;
0x501bd1;/usr/local/src/conda/python-3.9.19/Python/errors.c:327;MOV	%BPL,0x24(%R15);
0x501bd5;/usr/local/src/conda/python-3.9.19/Python/errors.c:419;MOV	0x38(%RSP),%RAX;
0x501bda;:0;SUB	%FS:0x28,%RAX;
0x501be3;:0;JNE	501cc9 <_PyErr_NormalizeException+0x229>;
0x501bf8;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:171;CALL	4ef470 <PyThreadState_Get>;
0x501bfd;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:73;MOV	0x8(%RBP),%R10;
0x501c01;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:171;MOV	%RAX,%R9;
0x501c04;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:74;TESTB	$0x8,0xa9(%R10);
0x501c0c;:0;JE	501c20 <_PyErr_NormalizeException+0x180>;
0x501c0e;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:81;MOV	0x38(%R10),%RDX;
0x501c12;:0;MOV	(%RBP,%RDX,1),%RAX;
0x501c17;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:114;TEST	%RAX,%RAX;
0x501c1a;:0;JNE	501cbc <_PyErr_NormalizeException+0x21c>;
0x501c20;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:116;XOR	%ECX,%ECX;
0x501c22;:0;XOR	%R8D,%R8D;
0x501c25;:0;XOR	%EDX,%EDX;
0x501c27;:0;MOV	%RBP,%RSI;
0x501c2a;:0;MOV	%R9,%RDI;
0x501c2d;:0;CALL	4f0370 <_PyObject_MakeTpCall>;
0x501c32;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:116;MOV	%RAX,%RCX;
0x501c35;:0;JMP	501b9f <_PyErr_NormalizeException+0xff>;
0x501c3a;/usr/local/src/conda/python-3.9.19/Python/errors.c:349;MOV	%RBP,%RSI;
0x501c3d;:0;MOV	%RDX,%RDI;
0x501c40;:0;MOV	%RDX,0x18(%RSP);
0x501c45;:0;CALL	514050 <PyObject_IsSubclass>;
0x501c4a;/usr/local/src/conda/python-3.9.19/Python/errors.c:350;TEST	%EAX,%EAX;
0x501c4c;:0;JS	501cfc <_PyErr_NormalizeException+0x25c>;
0x501c52;/usr/local/src/conda/python-3.9.19/Python/errors.c:360;MOV	0x18(%RSP),%RCX;
0x501c57;:0;JE	501b32 <_PyErr_NormalizeException+0x92>;
0x501c5d;/usr/local/src/conda/python-3.9.19/Python/errors.c:371;CMP	%RBP,%RCX;
0x501c60;:0;JNE	432099 <_PyErr_NormalizeException.cold+0xc7>;
0x501c66;/usr/local/src/conda/python-3.9.19/Python/errors.c:377;MOV	%RBP,(%RBX);
0x501c69;/usr/local/src/conda/python-3.9.19/Python/errors.c:378;MOV	%R13,(%R12);
0x501c6d;/usr/local/src/conda/python-3.9.19/Python/errors.c:379;MOVZX	0x24(%R15),%EBP;
0x501c72;/usr/local/src/conda/python-3.9.19/Python/errors.c:379;DEC	%EBP;
0x501c74;/usr/local/src/conda/python-3.9.19/Python/errors.c:380;JMP	501bd1 <_PyErr_NormalizeException+0x131>;
0x501c79;/usr/local/src/conda/python-3.9.19/Python/errors.c:97;XOR	%EDX,%EDX;
0x501c7b;:0;MOV	%R13,%RSI;
0x501c7e;:0;MOV	%RBP,%RDI;
0x501c81;:0;CALL	505730 <PyObject_Call>;
0x501c86;:0;MOV	%RAX,%RCX;
0x501c89;/usr/local/src/conda/python-3.9.19/Python/errors.c:97;JMP	501b9f <_PyErr_NormalizeException+0xff>;
0x501c8e;/usr/local/src/conda/python-3.9.19/Python/errors.c:97;MOV	%R11,0x18(%RSP);
0x501c93;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:118;XOR	%ECX,%ECX;
0x501c95;:0;MOV	%R14,%RDX;
0x501c98;:0;LEA	0x28(%RSP),%RSI;
0x501c9d;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:118;MOV	%RBP,%RDI;
0x501ca0;:0;CALL	%RAX;
0x501ca2;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:119;MOV	0x18(%RSP),%RDI;
0x501ca7;:0;XOR	%ECX,%ECX;
0x501ca9;:0;MOV	%RBP,%RSI;
0x501cac;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:118;MOV	%RAX,%RDX;
0x501caf;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:119;CALL	4f00f0 <_Py_CheckFunctionResult>;
0x501cb4;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:119;MOV	%RAX,%RCX;
0x501cb7;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:119;JMP	501b9f <_PyErr_NormalizeException+0xff>;
0x501cbc;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:119;MOV	%R9,0x18(%RSP);
0x501cc1;/usr/local/src/conda/python-3.9.19/Include/cpython/abstract.h:118;XOR	%ECX,%ECX;
0x501cc3;:0;XOR	%EDX,%EDX;
0x501cc5;:0;XOR	%ESI,%ESI;
0x501cc7;:0;JMP	501c9d <_PyErr_NormalizeException+0x1fd>;
0x501cc9;/usr/local/src/conda/python-3.9.19/Python/errors.c:419;CALLQ	0x20eab9(%RIP);
0x501ccf;/usr/local/src/conda/python-3.9.19/Python/errors.c:419;MOV	%RCX,0x18(%RSP);
0x501cd4;/usr/local/src/conda/python-3.9.19/Python/errors.c:104;MOV	0x244f7d(%RIP),%RDI;
0x501cdb;:0;XOR	%EAX,%EAX;
0x501cdd;:0;MOV	%RBP,%RDX;
0x501ce0;:0;MOV	0x18(%RSI),%RCX;
0x501ce4;/usr/local/src/conda/python-3.9.19/Python/errors.c:104;MOV	$0x614e68,%ESI;
0x501ce9;:0;CALL	4fe270 <PyErr_Format>;
0x501cee;/usr/local/src/conda/python-3.9.19/Include/object.h:422;MOV	0x18(%RSP),%R11;
0x501cf3;/usr/local/src/conda/python-3.9.19/Include/object.h:422;DECQ	(%R11);
0x501cf6;:0;JE	4320aa <_PyErr_NormalizeException.cold+0xd8>;
0x501cfc;/usr/local/src/conda/python-3.9.19/Include/object.h:422;DECQ	(%RBP);
0x501d00;:0;JE	4320db <_PyErr_NormalizeException.cold+0x109>;
0x501d06;/usr/local/src/conda/python-3.9.19/Include/object.h:422;DECQ	(%R13);
0x501d0a;:0;JE	4320ce <_PyErr_NormalizeException.cold+0xfc>;
0x501d10;/usr/local/src/conda/python-3.9.19/Python/errors.c:385;INCL	0x8(%RSP);
0x501d14;/usr/local/src/conda/python-3.9.19/Python/errors.c:385;MOV	0x8(%RSP),%EDI;
0x501d18;/usr/local/src/conda/python-3.9.19/Python/errors.c:386;CMP	$0x20,%EDI;
0x501d1b;:0;JE	432002 <_PyErr_NormalizeException.cold+0x30>;
0x501d21;/usr/local/src/conda/python-3.9.19/Python/errors.c:434;MOV	0x58(%R15),%RAX;
0x501d25;/usr/local/src/conda/python-3.9.19/Python/errors.c:395;MOV	0x10(%RSP),%R8;
0x501d2a;/usr/local/src/conda/python-3.9.19/Python/errors.c:438;PXOR	%XMM0,%XMM0;
0x501d2e;/usr/local/src/conda/python-3.9.19/Python/errors.c:395;MOV	(%R8),%RDI;
0x501d31;/usr/local/src/conda/python-3.9.19/Python/errors.c:434;MOV	%RAX,(%RBX);
0x501d34;/usr/local/src/conda/python-3.9.19/Python/errors.c:435;MOV	0x60(%R15),%R9;
0x501d38;/usr/local/src/conda/python-3.9.19/Python/errors.c:435;MOV	%R9,(%R12);
0x501d3c;/usr/local/src/conda/python-3.9.19/Python/errors.c:436;MOV	0x68(%R15),%R10;
0x501d40;/usr/local/src/conda/python-3.9.19/Python/errors.c:436;MOV	%R10,(%R8);
0x501d43;/usr/local/src/conda/python-3.9.19/Python/errors.c:440;MOVQ	$0,0x68(%R15);
0x501d4b;/usr/local/src/conda/python-3.9.19/Python/errors.c:438;MOVUPS	%XMM0,0x58(%R15);
0x501d50;/usr/local/src/conda/python-3.9.19/Python/errors.c:398;TEST	%RDI,%RDI;
0x501d53;:0;JNE	43204e <_PyErr_NormalizeException.cold+0x7c>;
0x501d59;/usr/local/src/conda/python-3.9.19/Python/errors.c:408;CMPL	$0x22,0x8(%RSP);
0x501d5e;:0;JNE	501ae1 <_PyErr_NormalizeException+0x41>;
